config.toml

```
@@ -35,21 +35,21 @@ title = "Global Hackathon | Hamburg"
     # Another "title" :-). This one is used as the site_name on the Hugo's internal
     # opengraph structured data template.
     # See https://ogp.me/ and https://gohugo.io/templates/internal#open-graph.
-    title = "Global Hackathon Hamburg"
+    title = "Global Hackathon TEMPLATE"
 
     # The "description" of your website. This is used in the meta data of your generated html.
     description = "Digital Earths Global Hackathon 12th to 17th May 2025"
 
     # A "copyright"-line to be added at the bottom of the page.
     # Hint: Plain HTML is supported in here as well. So you may e.g. add a link to your legal imprint-page.
-    copyright = "© nextGEMS project office"
+    copyright = "c  nextGES project office"
 
     # This theme will, by default, inject a design-by-line at the bottom of the page.
-    # You can turn it off, but we would really appreciate if you don’t :-)
+    # You can turn it off, but we would really appreciate if you don't :-)
     hidedesignbyline = false
 
     # The sections of the home page alternate styling. Mark invert as true to swap the styling of the sections
     invertSectionColors = false
 
     [params.meta]
-        keywords = "nextGEMS, WCRP, ESMO, hackathon, Digital Earths Global Hackathon, km-scale modelling, climate hackathon"
+        keywords = "template, WCRP, ESMO, hackathon, Digital Earths Global Hackathon, km-scale modelling, climate hackathon"
```

These are SEO keywords, I would remove "template" entirely and just leave the rest.
```suggestion
        keywords = "WCRP, ESMO, hackathon, Digital Earths Global Hackathon, km-scale modelling, climate hackathon"
```